web-ui/view/ref.ml

```diff
@@ -93,7 +99,7 @@ module Make (M : M_Git_forge) = struct
       let started_at = Timestamps_durations.pp_timestamp started_at in
       let ran_for = Timestamps_durations.pp_timestamp ran_for in
       row ~ref:(ref gref name) ~short_hash ~started_at ~ran_for ~status
-        ~ref_uri:(commit_url ~org ~repo short_hash)
+        ~ref_uri:(commit_url ~org ~repo hash)
```

Comment:
This breaks our convention of using long hashes in URLs. This fixes it and a subsequent PR seeks to harden this against future drift back to short hashes.

Reply:
I think that prior to introducing the new refs page, the old page used the full hash https://github.com/ocurrent/ocaml-ci/pull/565/files#diff-15d79b23f61af65eb663efd069be96a638f6c4ac6073879ddfd42a5bdd4f0437L50